lib/publickey.js

```diff
@@ -384,6 +385,15 @@ PublicKey.prototype.toBuffer = PublicKey.prototype.toDER = function() {
   }
 };
 
+/*
+ * Will return a sha256 + ripemd160 hash of the serialized public key
+ * @see https://github.com/bitcoin/bitcoin/blob/master/src/pubkey.h#L141
+ * @returns {Buffer}
+ */
+PublicKey.prototype.getID = function getID() {
```

Comment:
Hash has an entirely different meaning of a double sha256 hash, see: https://github.com/bitcoin/bitcoin/blob/master/src/pubkey.h#L148